packages/neuron-wallet/tests-e2e/wallet.test.ts

```diff
@@ -0,0 +1,224 @@
+import { Application } from 'spectron'
+import path from 'path'
+import { getElementByTagName, clickMenu, editWallet } from './utils'
+
+// Test create/import/switch/delete/edit wallet
+describe('wallet tests', () => {
+  let app: Application
+
+  beforeAll(() => {
+    let electronPath = path.join(__dirname, '..', 'node_modules', '.bin', 'electron')
+    if (process.platform === 'win32') {
+      electronPath += '.cmd'
+    }
+    app = new Application({ 
+      args: [
+        '--require',
+        path.join(__dirname, 'preload.js'),
+        path.join(__dirname, '..', 'dist', 'main.js')
+      ], 
+      path: electronPath 
+    })
+    return app.start()
+  })
+
+  afterAll(() => {
+    if (app.isRunning) {
+      return app.stop()
+    }
+  })
+
+  it('create wallet', async () => {
+    const { client } = app
+    await client.waitUntilWindowLoaded()
+
+    // Click create wallet
+    const createWalletButton = await getElementByTagName(client, 'button', '创建新钱包')
```

Comment:
Do we set language to Chinese explicitly or is this just an assumption which matches local development environment? If so we should use English as that's the default language (and as fallback language), and I believe our CI has that as default locale as well.

Reply:
Ok, I have added the locale parameter `--lang=en`. https://github.com/nervosnetwork/neuron/pull/852/commits/914890aa7154755da4316f4d3516dc6c14a3efba